qualtran/simulation/quimb_sim.py

```diff
@@ -120,6 +124,18 @@ def _assign_outgoing(cxn: Connection) -> Soquet:
         if pos is not None:
             fix[tuple([binst])] = pos[binst]
 
+    inds = get_right_and_left_inds(cbloq.signature)
+    if len(inds) != 2:
+        return tn, fix
+
+    l, r = inds
```

Comment:
Some comments here about what is going on here would be helpful (since this was a bug, it'd be good make a note).

Reply:
added comments